daemon/emer-persistent-cache.c

```diff
@@ -459,7 +459,13 @@ update_boot_offset (EmerPersistentCache *self,
         return save_timing_metadata (self, &relative_time, &absolute_time, NULL,
                                      NULL, NULL, error);
 
-      return TRUE;
+      /*
```

Comment:
The rest of the file uses a different style AFAICS so I will leave it like this unless you strongly object.